expressEngine/test/user/post/user.spec.js

```diff
@@ -0,0 +1,361 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import server from '../../../server';
+import { error, success } from '../../data';
+
+const { expect } = chai;
+const should = chai.should();
+chai.use(chaiHttp);
+
+describe('UserPost', () => {
+  describe('Post function in api/user/', () => {
+    it('register a user using given email and password', async () => {
+      const result = await chai
+        .request(server)
+        .post(success.userRoute)
+        .set(success.header)
+        .send({ email: success.email[0], password: success.password });
+      result.should.have.status(success.status);
+      result.body.should.have.property('data');
+      result.body.should.have.property('message');
+    });
+
+    describe('check for wrong email format', () => {
+      it('Without email field', async () => {
```

Comment:
`it('should fail without email field', async () => {` is better than `it('Without email field', async () => {`